firedrake/utility_meshes.py

```diff
@@ -554,18 +556,18 @@ def RectangleMesh(
 
     The boundary edges in this mesh are numbered as follows:
 
-    * 1: plane x == 0
+    * 1: plane x == originX
     * 2: plane x == Lx
-    * 3: plane y == 0
+    * 3: plane y == originY
     * 4: plane y == Ly
     """
 
     for n in (nx, ny):
         if n <= 0 or n % 1:
             raise ValueError("Number of cells must be a postive integer")
 
-    xcoords = np.linspace(0.0, Lx, nx + 1, dtype=np.double)
-    ycoords = np.linspace(0.0, Ly, ny + 1, dtype=np.double)
+    xcoords = np.linspace(originX, Lx, nx + 1, dtype=np.double)
+    ycoords = np.linspace(originY, Ly, ny + 1, dtype=np.double)
```

Comment:
This is now at odds with the docstring that says that the mesh has extent `Lx`/`Ly` (it now has extent `Lx - originX`, ...).

Reply:
Indeed, I've to fix that. Thank you !

Reply:
Not quite. Let's call the parameters the lower left and upper right corners of the box and then this works.